src/sentry/monitors/models.py

```diff
@@ -67,6 +67,10 @@ def get_next_schedule(last_checkin, schedule_type, schedule):
     else:
         raise NotImplementedError("unknown schedule_type")
 
+    # Ensure we clamp the expected time down to the minute, that is the level
+    # of granularity we're able to support
+    next_schedule = next_schedule.replace(second=0, microsecond=0)
```

Comment:
I was thinking we should put this on the `post_save` signal but that doesn't get fired when we update via queryset anyways I think. plus this will normalize `get_next_scheduled_checkin`